src/refdb_fs.c

```diff
@@ -844,19 +852,20 @@ static int refdb_fs_backend__lock(void **out, git_refdb_backend *_backend, const
 }
 
 static int refdb_fs_backend__write_tail(
+	git_filebuf *out_lock,
 	git_refdb_backend *_backend,
 	const git_reference *ref,
-	git_filebuf *file,
-	int update_reflog,
+	git__refdb_flags flags,
```

Comment:
As far as I know it's not allowed to use enums as bitmasks. You'd instead have to use e.g. an `unsigned int` here.